scripts/release/dev.env.dist

```diff
@@ -0,0 +1,17 @@
+# Value of 'x' will trigger downloading test file
+# This is actually the parent of the checkout dir without /molgenis-service-armadillo
+GIT_CHECKOUT_DIR=x
+DOWNLOAD_FILE=n
+DOWNLOAD_FILE_NAME=
+
+# path to file `gse66351_1.rda` including filename?
+RDA_TEST_DIR=
+
+PROFILE=xenon
+ARMADILLO_URL=http://localhost:8080
+ADMIN_PASSWORD=admin
+
+OIDC_EMAIL=your-email-is-invalid
+
+# The token will invalidate probably in 1 hour so replace it or set to empty
+TOKEN=
```

Comment:
For the entirety of the .env.dist file:

Set all variables to the `release-test.R` default, so for instance: 
- `GIT_CLONE_PATH=~/git`
- `RDA_FILE_PATH=~/git/molgenis-service-armadillo/data/testing`
- PROFILE, ARMADILLO_URL and ADMIN_PASSWORD are already set properly
- `OIDC_EMAIL=` (this has to be set by the user before running the tests)

With this, we no longer have to ask the user to enter anything anymore, just present the user with the variable set in the .env.dist config file, with a simple `enter to confirm` (if escape or so, stop running the code)

Reply:
I was on the CI testing flow.

Settings the defaults sound ok to me but I don't get `~git` and some. We'll've to discuss. I tried in the code using FIXME.

We could add `ASK_FOR_CONFIRMATION=N` but no sure as you have to copy the `dev.env.dist` to `dev.env` or use environment variables to tune this.

I think reading from `dev.env` should actually be `.env` following the dot-dev way. But it is hidden then :-O but CI don't care I guess

Reply:
We discussed:
- `*.env files: users have to copy one of the `*.env.dist` files to `.env`. Currently we have one but expect another like `tst.env.dist` or `stage.env.dist` and `ci.env.dist` ... what are [dot env files](https://hexdocs.pm/dotenvy/dotenv-file-format.html)
- as our users have a git checkout most of the time we can stream-line path and files removing Y/N questions